src/options/smt_options.toml

```diff
@@ -634,7 +634,7 @@ header = "options/smt_options.h"
 [[option]]
   name       = "solveRealAsInt"
   category   = "undocumented"
-  long       = "solve-real-as-int"
+  long       = "solve-real-as-int=N"
```

Comment:
This is bool :)

Reply:
I added a check for bool/void options that aborts if `=...` is specified.